nextflow.config

```diff
@@ -19,6 +19,12 @@ params {
     igenomes_base              = 's3://ngi-igenomes/igenomes/'
     igenomes_ignore            = false
 
+    // Kraken2 options
+    kraken2_db                 = 'https://github.com/nf-core/test-datasets/raw/taxprofiler/data/database/kraken2/testdb-kraken2.tar.gz'
+    kraken2_save_reads         = false
+    kraken2_save_readclassifications = false
+    save_uncompressed_k2db     = false
```

Comment:
oki, maybe you need to use this flag to write a condition in the modules.conf to custom for the process the output of this file in the publishDir or remove the general statement. No idea which option is preferred